docs/management/managing-indices.asciidoc

```diff
@@ -13,7 +13,12 @@ the amount of bookkeeping when working with indices. Instead of manually
 setting up your indices, you can create them automatically from a template,
 ensuring that your settings, mappings, and aliases are consistently defined.
 
-To manage your indices, open the menu, then go to *Stack Management > Data > Index Management*.
+You can also use *Index Management* to view and manage
```

Comment:
There's a lot of content in this topic.  How about rewriting the intro with bullet points about "what you'll learn"?